src/rust/src/rdataframe/read_parquet.rs

```diff
@@ -6,6 +6,25 @@ use extendr_api::Rinternals;
 use extendr_api::{extendr, extendr_module, Robj};
 use polars::io::RowIndex;
 use polars::prelude::{self as pl};
+
+pub fn robj_to_cloudoptions<'a>(
```

Comment:
Can you put this function in `rdatatype.rs` with the other `robj_to_*()` functions? I just noticed that those are split across a couple of files so this should be harmonized later

Also, can you name this `robj_to_cloud_options`?